pandas/io/tests/test_sas.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+import pandas.util.testing as tm
+from pandas import compat
+from pandas.io.sas import XportReader, read_sas
+import os
+
+# CSV versions of test XPT files were obtained using the R foreign library
+
+class TestXport(tm.TestCase):
+
+    def setUp(self):
+        self.dirpath = tm.get_data_path()
+        self.file01 = os.path.join(self.dirpath, "DEMO_G.XPT")
+        self.file02 = os.path.join(self.dirpath, "SSHSV1_A.XPT")
+        self.file03 = os.path.join(self.dirpath, "DRXFCD_G.XPT")
+
+
+    def test1(self):
+        # Tests with DEMO_G.XPT (all numeric file)
+
+        # Compare to this
+        data_csv = pd.read_csv(self.file01.replace(".XPT", ".csv"))
+
+        # Read full file
+        data = XportReader(self.file01).read()
+        tm.assert_frame_equal(data, data_csv, check_dtype=False)
```

Comment:
instead of using ``check_dtype=False``, change the ``data_csv`` dtypes to match what  you expect